apps/src/javalab/JavalabView.jsx

```diff
@@ -114,7 +116,35 @@ class JavalabView extends React.Component {
     this.props.appendOutputLog('Compiled!');
   };
 
+  renderSettings = () => {
+    const {isDarkMode} = this.props;
+    return [
+      <a onClick={this.props.toggleDarkMode}>
+        Switch to {isDarkMode ? 'light mode' : 'dark mode'}
+      </a>
+    ];
+  };
+
+  getButtonStyles = isSettings => {
```

Comment:
Super minor NIT: I didn't understand what `isSettings` was until I read how `getButtonStyles` was used. Perhaps `isSettingsButton` would be clearer?